storage_scrubber/src/checks.rs

```diff
@@ -102,8 +106,16 @@ pub(crate) fn branch_cleanup_and_check_errors(
                     }
 
                     if index_part.layer_metadata.is_empty() {
-                        // not an error, can happen for branches with zero writes, but notice that
-                        info!("index_part.json has no layers");
+                        if index_part.metadata.ancestor_timeline().is_none() {
+                            // The initial timeline with no ancestor should ALWAYS have layers.
+                            result.errors.push(
```

Comment:
also logging it?

Reply:
The scrubber does it here:
https://github.com/neondatabase/neon/blob/3773a945f517a2d451397cc25bd887a710ef8574/storage_scrubber/src/checks.rs#L182